src/User32.Desktop/User32+SafeDCHandle.cs

```diff
@@ -26,8 +26,12 @@ public class SafeDCHandle : SafeHandle
             /// </summary>
             /// <param name="hWnd">The HWND this handle is associated with and must be released with.</param>
             /// <param name="hDC">The handle to the DC.</param>
-            public SafeDCHandle(IntPtr hWnd, IntPtr hDC)
-                : base(IntPtr.Zero, true)
+            /// <param name="ownsHandle">
+            ///     <see langword="true" /> to have the native handle released when the
+            ///     <see cref="SafeDCHandle" /> is disposed or finalized; <see langword="false" /> otherwise.
```

Comment:
I would replace the reference to `SafeDCHandle` to just "this instance" or "this safe handle". That way, it's more copy and paste-able across types.